packages/typespec-python/src/types.ts

```diff
@@ -302,20 +304,36 @@ function emitConstant(type: SdkConstantType) {
     });
 }
 
+function emitEnumValueType(type: SdkEnumValueType): Record<string, any> {
+    return getSimpleTypeResult({
+        type: "constant",
+        value: type.value,
+        valueType: emitBuiltInType(type),
+    });
+}
+
 const sdkScalarKindToPythonKind: Record<string, string> = {
     int32: "integer",
     int64: "integer",
     float32: "float",
     float64: "float",
 };
 
-function emitBuiltInType(type: SdkBuiltInType | SdkDurationType | SdkDatetimeType): Record<string, any> {
+function emitBuiltInType(
+    type: SdkBuiltInType | SdkDurationType | SdkDatetimeType | SdkEnumValueType,
+): Record<string, any> {
     if (type.kind === "duration" && type.encode === "seconds") {
         return getSimpleTypeResult({
             type: sdkScalarKindToPythonKind[type.wireType.kind],
             format: type.encode,
         });
     }
+    if (type.kind === "enumvalue") {
+        return getSimpleTypeResult({
+            type: "string",
+            format: "string",
```

Comment:
Here is hard code since discriminator must be `string`
![image](https://github.com/Azure/autorest.python/assets/70930885/203bc7ab-9aa2-4630-b4d8-8ea063264711)